coral/src/app/pages/approvals/index.tsx

```diff
@@ -7,6 +7,7 @@ import {
   isApprovalsTabEnum,
 } from "src/app/router_utils";
 import ApprovalResourceTabs from "src/app/features/approvals/ApprovalResourceTabs";
+import AuthenticationRequiredBoundary from "src/app/components/AuthenticationRequiredBoundary";
```

Comment:
Very minor, but I feel this commit does not belong into this PR. It's not related to the scope really. Could we maybe move that change to a later PR?